frontend/components/ClickableUrls/ClickableUrls_xss.tests.tsx

```
@@ -0,0 +1,59 @@
+      <ClickableUrls text='Check <script>alert("xss")</script> this site' />
+    );
+    expect(container.innerHTML).not.toContain("<script>");
+    expect(container.innerHTML).not.toContain("alert");
+  });
+
+  it("strips javascript: protocol when injected via HTML anchor", () => {
+    // Plain text "javascript:" is harmless -- the risk is only when it
+    // appears inside an href attribute. DOMPurify should strip it there.
+    const malicious =
+      // eslint-disable-next-line no-script-url
+      'Click <a href="javascript:alert(\'xss\')">here</a> for details';
```

ESLint

[error] Replace `'Click·<a·href="javascript:alert(\'xss\')">here</a>·for·details'` with `"Click·<a·href=\"javascript:alert('xss')\">here</a>·for·details"`

(prettier/prettier)